ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigPhase0.java

```diff
@@ -224,7 +226,8 @@ public SpecConfigPhase0(
       final Bytes4 electraForkVersion,
       final UInt64 electraForkEpoch,
       final Bytes4 fuluForkVersion,
-      final UInt64 fuluForkEpoch) {
+      final UInt64 fuluForkEpoch,
+      final boolean blsDisabled) {
```

Comment:
Maybe we can have a constructor without the `blsDisabled` parameter, and one constructor overload with the parameter and annotated with `@VisibleForTests` or something to make it clear that it is only used on tests?

This way we also don't need to change existing calls to the constructor.

Reply:
We can also add a unit test to check that the default constructor (without the paramater) will always create a spec with blsDisabled = false.